client/core/core.go

```diff
@@ -1700,6 +1700,7 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 	if err != nil {
 		return err
 	}
+	defer crypter.Close()
```

Comment:
Hmm, we don't do this *anywhere*, and there seem to be numerous places we should.  Can we split this out and address them all in another PR?

Reply:
ok.

Reply:
diff https://github.com/decred/dcrdex/pull/1582/commits/0fdbd4e6688a9ac47b0812c22ede529f297575bc